docs/src/dev-docs/design-docker-compose.md

```diff
@@ -1,199 +0,0 @@
-  db_table_creator -->|completed_successfully| query_scheduler
-  queue -->|healthy| query_scheduler
-  redis -->|healthy| query_scheduler
-  query_scheduler -->|healthy| reducer
-  results_cache_indices_creator -->|completed_successfully| reducer
-  db_table_creator -->|completed_successfully| webui
-  results_cache_indices_creator -->|completed_successfully| webui
-  db_table_creator -->|completed_successfully| garbage_collector
-  results_cache_indices_creator -->|completed_successfully| garbage_collector
-
-  subgraph Databases
-    database
-    queue
-    redis
-    results_cache
-  end
-
-  subgraph DB Migration Jobs
-    db_table_creator
-    results_cache_indices_creator
-  end
-
-  subgraph Schedulers
-    compression_scheduler
-    query_scheduler
-  end
-
-  subgraph Workers
-    compression_worker
-    query_worker
-    reducer
-  end
-
-  subgraph UI & Management
-    webui
-    garbage_collector
-  end
-:::
-
-### Services overview
-
-The CLP package is composed of several service components. The tables below list the services and
-their functions.
-
-:::{table} Services
-:align: left
-
-| Service               | Description                                                     |
-|-----------------------|-----------------------------------------------------------------|
-| database              | Database for archive metadata, compression jobs, and query jobs |
-| queue                 | Task queue for schedulers                                       |
-| redis                 | Task result storage for workers                                 |
-| compression_scheduler | Scheduler for compression jobs                                  |
-| query_scheduler       | Scheduler for search/aggregation jobs                           |
-| results_cache         | Storage for the workers to return search results to the UI      |
-| compression_worker    | Worker processes for compression jobs                           |
-| query_worker          | Worker processes for search/aggregation jobs                    |
-| reducer               | Reducers for performing the final stages of aggregation jobs    |
-| webui                 | Web server for the UI                                           |
-| garbage_collector     | Background process for retention control                        |
-:::
-
-### One-time initialization jobs
-
-We also set up short-lived run-once "services" to initialize some services listed above.
-
-:::{table} Initialization jobs
-:align: left
-
-| Job                           | Description                                             |
-|-------------------------------|---------------------------------------------------------|
-| db-table-creator              | Initializes database tables                             |
-| results-cache-indices-creator | Initializes single-node replica set and sets up indices |
-:::
-
-## Troubleshooting
-
-If you encounter issues with the Docker Compose deployment:
-
```

Comment:
shall we also tell users how to check the instance id so they can deduct the project name? (or maybe list all docker compose projects with `docker compose ls` to tell what's running?

Reply:
Yeah, good idea.